seasonofdocs/README.md

```diff
@@ -1,98 +1,28 @@
----
-maintainers:
-- caniszczyk
-- idvoretskyi
-- zacharysarah
-- celestehorgan
----
-
-# Google Season of Docs 2021
-
-Google [Season of Docs](https://developers.google.com/season-of-docs) (GSoD) gives technical writers an opportunity to work with open source projects.
-
-The CNCF is applying to participate in GSoD 2021!
-
-## 2021 Timeline
-
-Google Season of Docs 2021 makes numerous change to the program from previous years:
-
-1. GSoD 2021 permits only one project proposal per organization. 
-
-   This marks a significant change from the multiple proposals allowed in previous years. 
-
-1. GSoD 2021 starts earlier and runs longer! Projects begin in April/May and run until November.
-
-1. Project proposals include a new requirement for budgeting.
-
-   For more information, see the [proposal template](https://developers.google.com/season-of-docs/docs/org-proposal-template).
-
-### What's happening right now?
-
-Date | Activity
----|---
-March 26, 2021 | The CNCF has submitted a [project proposal](#project-proposal) for consideration.
+# Season of Docs
 
-All steps are based on the [GSoD 2021 timeline](https://developers.google.com/season-of-docs/docs/timeline).
+Google [Season of Docs](https://developers.google.com/season-of-docs) gives technical writers an opportunity to work with open source projects.
 
-## Project Proposal
+The CNCF is applying to participate in the Season of Docs as an org in 2023. We'll announce more details pending our acceptance into the program.
```

Comment:
@nate-double-u  Hi Nate, when will GSoD 2023 be opened for CNCF maintainers to submit the project proposals?

Reply:
Sorry i was unclear, you can submit proposals now. Here's the announcement: https://github.com/cncf/mentoring/discussions/843